behaviors/TwoHandViewManipBehavior.cs

```diff
@@ -147,12 +147,12 @@ public override Capture BeginCapture(InputState input, CaptureSide eSide)
             bool bHit1 = cockpit.Scene.FindSceneRayIntersection(input.vLeftSpatialWorldRay, out hit1);
             bool bHit2 = cockpit.Scene.FindSceneRayIntersection(input.vRightSpatialWorldRay, out hit2);
             if ( bHit1 && bHit2 ) {
-                Vector3 avg = (hit1.hitPos + hit2.hitPos) * 0.5f;
-                float d = VRUtil.GetVRRadiusForVisualAngle(avg, camFrame.Origin, 2.0f);
+                Vector3 avg = ((hit1.hitPos + hit2.hitPos) * 0.5f).ToVector3();
```

Comment:
No point in casting to Vector3 here -- all usages re-cast avg to Vector3f anyway.